aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -140,6 +140,97 @@ public class LogBufferDescriptor
      */
     public static final int LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH = CACHE_LINE_LENGTH * 2;
 
+    /**
+     * Offset within the log metadata where the sparse property is stored.
+     */
+    public static final int LOG_IS_SPARSE_OFFSET;
+
+    /**
+     * Offset within the log metadata where the tether property is stored.
+     */
+    public static final int LOG_IS_TETHER_OFFSET;
+
+    /**
+     * Offset within the log metadata where the rejoin property is stored.
+     */
+    public static final int LOG_IS_REJOIN_OFFSET;
+
+    /**
+     * Offset within the log metadata where the reliable property is stored.
+     */
+    public static final int LOG_IS_RELIABLE_OFFSET;
```

Comment:
`IS_` should be removed, i.e. field is called `reliable`.

Reply:
Done